src/components/nav_drawer/nav_drawer.tsx

```diff
@@ -177,14 +244,14 @@ export class EuiNavDrawer extends Component {
         // does not allow for deep `ref` element management at present
         const element = document.querySelector(
           `#${MENU_ELEMENT_ID} [${ATTR_SELECTOR}='${focusReturn}']`
-        );
+        ) as HTMLInputElement;
```

Comment:
There is not a guarantee that the element is an `input`, target `HTMLElement` instead.

```suggestion
        ) as HTMLElement;
```